app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,13 @@
+doctype html
+html
+  head
+    title Ruby Google Scraper
```

Comment:
I think we should add the I18n support for the title as well.
Normally, the title will be shown as `Ruby Google Scraper` on the homepage, and `Ruby Google Scraper | another page` on other pages

Reply:
Fixed on 05249c82560b142a7dc772574c66633d750186ff d8f50c0d0975ffa876e3be34a47b190b24c645bb